HospitalLibrary/HospitalLibrary/Patient.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HospitalLibrary
+{
+    public class Patient
+    {
+        public string Name;
+        public string Surname;
+        public override string ToString()
+        {
+            return $"{Name} {Surname}";
+        }
+
+        public readonly int healthInsurance;
```

Comment:
Открытое (public) свойство лучше называть с заглавной буквы:
```suggestion
        public readonly int HealthInsurance;
```